gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/FirestoreDataStore.java

```diff
@@ -49,7 +49,7 @@
   // TODO(#57): Set to false to disable Firebase-managed offline persistence once local db sync
   // is implemented for project config and users.
   private static final FirebaseFirestoreSettings FIRESTORE_SETTINGS =
-      new FirebaseFirestoreSettings.Builder().setPersistenceEnabled(true).build();
+      new FirebaseFirestoreSettings.Builder().setPersistenceEnabled(false).build();
```

Comment:
W00t! Congratulations! 🚀

Reply:
I forgot to enable it back after testing. But I don't think we need it anymore. Also, keeping the persistence disabled will help us test the local db with daily usage.